test/helpers/capture-log-output.ts

```diff
@@ -8,7 +9,7 @@ export async function captureLogOutput(
   let outputData = "";
 
   // @ts-expect-error
-  let stdoutSpy = jest.spyOn(log[streamSym], "write") as jest.SpyInstance;
+  let stdoutSpy = vi.spyOn(log[pinoSymbols.streamSym], "write") as SpyInstance;
```

Comment:
on second thought this seems cleaner.

```suggestion
  const stdoutSpy: SpyInstance = vi.spyOn(
    // @ts-expect-error
    log[streamSym],
    "write",
  );
```